plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineSdkClasspathContainer.java

```diff
@@ -8,17 +8,14 @@
 import org.eclipse.jdt.core.IClasspathEntry;
 import org.eclipse.jdt.core.JavaCore;
 
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
 import com.google.cloud.tools.eclipse.sdk.CloudSdkProvider;
 
 public final class AppEngineSdkClasspathContainer implements IClasspathContainer {
 
-  // TODO should be changed once app-tools-lib can provide the directory
-  // https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/issues/134
-  private static final String SDK_JAR = "platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar";
+  private static final CloudSdk CLOUD_SDK = new CloudSdkProvider().getCloudSdk();
```

Comment:
This shouldn't be a static.  Actually, `AppEngineSdkClasspathContainer` should accept being provided with a `CloudSdk` instance.